src/components/PaginationFooter/DisplayedRowsLabel.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { StyledTypography } from './styles';
+import { usePaginationContext } from './context/PaginationProvider';
+
+const DisplayedRowsLabel = () => {
+  const { translator, width, pageInfo, totalRowCount } = usePaginationContext();
+  const { page, rowsPerPage } = pageInfo;
+
+  if (width <= 250) {
+    return null;
+  }
+
+  const displayedRowsText = translator.get('NebulaTableUtils.Pagination.DisplayedRowsLabel', [
```

Comment:
useMemo around this one would be optimal IMO
I just remember that it may be lots of rerenders in PVT on the same page

Or if we come one step above this, maybe it would be better to memoise some parts of Pagination component in general